lib/plenario_aot/aot_actions.ex

```diff
@@ -0,0 +1,153 @@
+defmodule PlenarioAot.AotActions do
+  require Logger
+
+  import Ecto.Query
+
+  alias Plenario.Repo
+
+  alias PlenarioAot.{AotMeta, AotData, AotObservation}
+
+  def create_meta(network_name, source_url) do
+    params = %{network_name: network_name, source_url: source_url}
+    meta = %AotMeta{}
+
+    Logger.info("Creating new AoT Meta with params `#{inspect(params)}`")
+
+    AotMeta.changeset(meta, params)
+    |> Repo.insert()
+  end
+
+  def list_metas, do: Repo.all(AotMeta)
+
+  def get_meta(identifier) when is_integer(identifier), do: Repo.get_by(AotMeta, id: identifier)
+  def get_meta(identifier) when is_bitstring(identifier), do: Repo.get_by(AotMeta, slug: identifier)
+
+  def update_meta(%AotMeta{} = meta, params \\ []) do
```

Comment:
This makes it easier to update metadata -- we just pass it a keyword list and avoid having to grep through a map:

```elixir
update_meta(meta, bbox: new_bbox)
```